bitnami/harbor/templates/chartmuseum/chartmuseum-dpl.yaml

```diff
@@ -74,17 +100,35 @@ spec:
             timeoutSeconds: {{ .Values.chartmuseum.readinessProbe.timeoutSeconds }}
             successThreshold: {{ .Values.chartmuseum.readinessProbe.successThreshold }}
             failureThreshold: {{ .Values.chartmuseum.readinessProbe.failureThreshold }}
+          {{- else if .Values.chartmuseum.customReadinessProbe }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.chartmuseum.customReadinessProbe "context" $) | nindent 12 }}
           {{- end }}
           envFrom:
             - configMapRef:
                 name: {{ include "harbor.chartmuseum" . }}-envvars
             - secretRef:
```

Comment:
Just to understand, you removed checksum/secret but you keep it as ref here, shouldn't it be removed from here or keep it in both?

Reply:
Agree, restored